profiles/default

```diff
@@ -19,13 +18,12 @@ set rules {
   T008
   T009
   T010
-  T011
-  T013
```

Comment:
Is there a reason to not enforce this?

Reply:
I mean the copyright notice

Reply:
Copyright notices have a pretty doubtful usefulness, given that
- they're rarely kept up to date
- in a lot of places (France and the US being some of them), the actual copyright is the one who wrote the code unless there's an explicit contract which says otherwise.

AFAIK, the copyright notices that were present in american open-source code for a long time was there because the american copyright law required it. This is not the case anymore even in the US (and was never the case in e.g. France)

I am keen to keep the copyright information in the README and manifest.xml up to date as a documentation, but people have to keep in mind that it is not authoritative. Git is actually a better source of copyright information than any notice.

Reply:
Fair enough